### PR TITLE
Rel 1.0.0 what to expect in net7

### PR DESCRIPTION
**Pull request by jamattey**

### CODE REVIEW

File: Demos/Blazorise.Demo.AntDesign/Properties/launchSettings.json [hunk 1]
```diff
@@ -20,10 +20,10 @@
       "commandName": "Project",
       "launchBrowser": true,
       "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
+      "applicationUrl": "http://localhost:7715/",
```
Comment (stsrki) [inline Demos/Blazorise.Demo.AntDesign/Properties/launchSettings.json:23]: Why did you change all the `launchSettings.json` files?

File: Demos/Blazorise.Demo.Material/Blazorise.Demo.Material.csproj [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
   <Import Project="..\..\Build\Blazorise.Client.props" />
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Blazorise.Demo.Material.Client' ">
+    <ExternalConsole>true</ExternalConsole>
```
Comment (stsrki) [inline Demos/Blazorise.Demo.Material/Blazorise.Demo.Material.csproj:6]: Remove this changes from all `.csproj` files.

File: Documentation/Blazorise.Docs.Server/Blazorise.Docs.Server.csproj [hunk 3]
```diff
@@ -11,4 +11,10 @@
     <ProjectReference Include="..\Blazorise.Docs\Blazorise.Docs.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Remove="wwwroot\img\blog\2022-08-23\" />
```
Comment (stsrki) [inline Documentation/Blazorise.Docs.Server/Blazorise.Docs.Server.csproj:15]: Remove this ItemGroup's

File: Tests/BasicTestApp.Server/BasicTestApp.Server.csproj [hunk 4]
```diff
@@ -5,6 +5,7 @@
     <LangVersion>10.0</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'Blazorise.BasicTestApp.Server' " />
```
Comment (stsrki) [inline Tests/BasicTestApp.Server/BasicTestApp.Server.csproj:8]: Remove this also.

File: Documentation/Blazorise.Docs/Pages/Blog/2022-08-23_WhatToExpectInNET7/Index.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+
+![HTTP 3](img/blog/2022-08-23/Http3.png align="left")
+
+### Minimal APIs Improvements
+Minimal APIs were introduced in .Net 6 to allow you to create lightweight APIs without the overhead of controllers. These APIs have seen massive improvements with the introduction of the new Microsoft.AspNetCore.OpenApi package to provide APIs for interacting with the OpenAPI specification in minimal APIs. 
+
+Also, endpoint filters will allow you to implement crosscutting concerns that you can only do with controllers using action filters today.
+
+## Should You Migrate To .NET 7?
```
Comment (stsrki) [inline Documentation/Blazorise.Docs/Pages/Blog/2022-08-23_WhatToExpectInNET7/Index.md:95]: Mention that we will in Blazorise 1.1 support .NET7 and in time we will slowly add features like `required` keyword, INumber interfaces for generic components like NumericEdit and NumericPicker, etc.

For now we will only do the NET7 support.

File: Demos/Blazorise.Demo.AntDesign/Properties/launchSettings.json (hunk 1 above)
Comment (jamattey) [reply Demos/Blazorise.Demo.AntDesign/Properties/launchSettings.json:23]: I compiled it like i usually do. I do not touch any other files
